src/components/Input/Input.tsx

```diff
@@ -219,6 +221,31 @@ class Input extends AutoControlledComponent<Extendable<IInputProps>, any> {
       </ElementType>
     )
   }
+  */
+
+  renderComponent({ ElementType, classes, styles, variables, rest }) {
+    const { input, onChange } = this.props
+    const inputClasses = classes.input
+
+    return (
+      <ElementType className={classes.root} onChange={onChange} {...rest}>
+        {createHTMLInput(input, {
+          overrideProps: {
+            className: inputClasses,
+            ref: this.handleInputRef,
+          },
+        })}
+        {this.computeIcon() &&
```

Comment:
note, `renderIcon` support is lost with suggested changes

Reply:
We can iterate again on the Input once the new PR for it is merged, and can see if there is any other issues. This are just temporary fixes anyway.